datamigration/codec.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package datamigration;
+
+import "codec.proto";
+import "gogoproto/gogo.proto";
+
+message ExecutedMigration {
+  weave.Metadata metadata = 1;
```

Comment:
Can you explain to me why there is no data here? How this model represents an executed migration?

Reply:
There is no data that I need to store. I only need to keep an information that a migration was executed. Value of the stored pair does not matter.